src/js/services/srv-asset.js

```diff
@@ -0,0 +1,12 @@
+const app = angular.module('materia')
+app.service('AssetSrv', function () {
+	const deleteAsset = (asset) =>
+		Materia.Coms.Json.del(`/api/asset/delete/${encodeURIComponent(asset.id)}`, asset)
+	const restoreAsset = (asset) =>
+		Materia.Coms.Json.del(`/api/asset/restore/${encodeURIComponent(asset.id)}`, asset)
```

Comment:
This will generate a `DELETE` request to `/api/asset/restore` which... doesn't really make sense. `Materia.Coms.Json.post` would make more sense here.